api/resources/translations/messages-ne.properties

```diff
@@ -766,6 +766,7 @@ messages.f.report_accepted_parent =
 messages.generic.no_provided_patient_id = 
 messages.generic.provided_patient_id_not_unique = तपाइँले पठाउनुभएको आईडी पहिल्यै प्रयोगमा छ।  
 messages.generic.provided_phone_not_valid = तपाइँले पठाउनुभएको फोन नम्बर मिलेन । फेरी प्रयास गर्नुहोस्।
+messages.invalid_phone= तपाइँले पठाउनुभएको फोन नम्बर मिलेन । फेरी प्रयास गर्नुहोस्।
```

Comment:
Same value for these two, can we use same key ?

Reply:
I saw in your app settings you had added invalid_phone hence I thought it was used somewhere. (i.e. Became a sheep) Will remove it.